src/Nest/CommonOptions/DateMath/DateMath.cs

```diff
@@ -79,10 +79,11 @@ public static DateMath FromString(string dateMath)
 			return math;
 		}
 
+		internal bool IsValid => Self.Anchor.Match(d => d != default, s => !s.IsNullOrEmpty());
+
 		public override string ToString()
 		{
-			var isValid = Self.Anchor.Match(d => d != default(DateTime), s => !s.IsNullOrEmpty());
-			if (!isValid) return string.Empty;
+			if (!IsValid) return string.Empty;
```

Comment:
Should we have a separate `DateMath` test for this?

Reply:
We should! Attached a unit test for this.